numba/cuda/decorators.py

```diff
@@ -129,4 +129,8 @@ def autojitwrapper(func):
 
 def declare_device(name, sig):
```

Comment:
Side note... should this have a docstring, it's public under `cuda`?

Reply:
It should, and there's one added here in #7621 - this patch is just to fix #7698: https://github.com/numba/numba/pull/7621/files#diff-0359579dfbc7075ce20946e2fcc1d62aefd21d177133a0821d5666c460ccc28aL130

Reply:
ah great, thanks for adding that.